lib/handlers.js

```diff
@@ -75,6 +75,7 @@ exports.put = function(req, res, next) {
 
   // Update in 1 atomic operation on the database if not specified otherwise
   if (this.shouldUseAtomicUpdate) {
+    this.update_options.new = true;
```

Comment:
It's not a matter of which one is better. This change forces all users of the node-restful library to get the modified object.

There are use cases where getting the unmodified document is better -- this change would prevent users with that use case from doing what they want